docs/overview/visual-cpp-language-conformance.md

```diff
@@ -208,17 +208,18 @@ For details on conformance improvements, see [C++ conformance improvements in Vi
 | &emsp;[`P2316R2 Consistent character literal encoding`](https://wg21.link/p2316r2) | VS 2022 17.0 <sup>[23](#note_23)</sup> |
 | &emsp;[`P2324R2 Labels at the end of compound statements (C compatibility)`](https://wg21.link/p2324r2) | no |
 | &emsp;[`P2327R1 De-deprecating volatile compound operations`](https://wg21.link/p2327r1) | no |
-| &emsp;[`P2334R1 preprocessing directives elifdef and elifndef`](https://wg21.link/p2334r1) | no |
+| &emsp;[`P2334R1 preprocessing directives elifdef and elifndef`](https://wg21.link/p2334r1) | VS 2022 17.10 <sup>[23](#note_23)</sup> |
 | &emsp;[`P2360R0 Extend init-statement to allow alias-declaration`](https://wg21.link/p2360r0) | no |
 | &emsp;[`P2362R3 Remove non-encodable wide character literals and multicharacter wide character literals`](https://wg21.link/p2362r3) | no |
 | &emsp;[`P2437R1 Support for #warning`](https://wg21.link/p2437r1) | no |
 | &emsp;[`P2448R2 Relaxing some constexpr restrictions`](https://wg21.link/p2448r2) | no |
 | &emsp;[`P2460R2 Relax requirements on wchar_t to match existing practices`](https://wg21.link/p2460r2) | no |
 | &emsp;[`P2468R2 The Equality Operator You Are Looking For`](https://wg21.link/p2468r2) | no |
-| &emsp;[`P2493R0 Missing feature test macros for C++20 core papers`](https://wg21.link/p2493r0) | no |
+| &emsp;[`P2493R0 Missing feature test macros for C++20 core papers`](https://wg21.link/p2493r0) | VS 2022 17.2 <sup>[DR](#note_DR)</sup> |
```

Comment:
Seems that P2493R0 is a DR, hence the change of superscript in commit https://github.com/MicrosoftDocs/cpp-docs/commit/14e1a74b8980973eb849ed65a36f27026098fd53. However, looking at the definition of DR:

https://github.com/MicrosoftDocs/cpp-docs/blob/25815cbe0dd6bae018890dd85eb10e5e5feece8e/docs/overview/visual-cpp-language-conformance.md?plain=1#L646

It does not seem to be applicable to this and a couple of other usages.